src/reports/models/statsManager.js

```diff
@@ -31,6 +32,17 @@ module.exports.postStats = async (report, stats) => {
     return stats;
 };
 
+async function updateSubscriberWithStatsInternal(report, stats) {
+    const parseData = JSON.parse(stats.data);
+    const subscriber = report.subscribers.find(subscriber => subscriber.runner_id === stats.runner_id);
+    const { last_stats } = subscriber;
+    if (last_stats && parseData.rps) {
+        const lastTotalCount = _.get(last_stats, 'rps.total_count') || 0;
```

Comment:
```suggestion
        const lastTotalCount = _.get(last_stats, 'rps.total_count', 0);
```

see https://lodash.com/docs/4.17.15#get

Reply:
cool